README.md

```diff
@@ -24,8 +24,8 @@ the _API integration_ page.
 ```javascript
 notifyClient
 	.sendSms(templateId, phoneNumber, personalisation, reference)
-	.then((response) => { })
-	.catch((err) => {})
+	.then(response => console.log)
```

Comment:
Yeah, you’re right. Changed it to the latter one cause I think it’s more explicit/easier to adapt.